src/mmw/js/src/modeling/models.js

```diff
@@ -400,9 +413,29 @@ var ScenarioModel = Backbone.Model.extend({
                 }
             };
 
+        self.updateInputHash();
+        self.attemptSave();
         taskModel.start(taskHelper);
     },
 
+    updateInputHash: function() {
+        var standardizedInputs = this.get('inputs').sortBy('name').map(function(inputModel) {
+            var input = inputModel.toJSON(),
+                sortedInput = {};
```

Comment:
I don't think this implementation is correct, but the idea looks right.  The result here is an array with one element which is essentially the same object that was started with `[{name: "", shape: "", etc...]`, I think we really want an array with key/value pairs `[{name: ""}, {shape: ""}, etc..]` since array index order is guaranteed.  I believe we were avoiding stringifying on a multi-attributed object since key order was not assured.